web/src/app/validation/validation-util.ts

```diff
@@ -19,4 +19,8 @@ export class ValidationUtil {
         }
         return true;
     }
+
+    public static compareStrTrimed(s1: string, s2: string): boolean {

```

Comment:
There should be undefined/null checks in utility functions like this one. Please add.